### ISSUE TITLE
Option noEmit doesn't work

### ISSUE THREAD

**Issue by hhyyrylainen** [baptistedonaux/brunch-typescript#26]
I was getting this error while trying to build typescript files: `error: Compiling of app/drawable.ts failed. Debug Failure. False expression: Output generation failed`

Looks like if I put `"noEmit": true,` in my tsconfig.json, to prevent my error checker from generating any files, brunch-typescript fails to override that. 
Even if I put this in my brunch-config.coffee:
```
plugins:
    brunchTypescript:
        noEmit: false
        removeComments: true
        watch: false
```

I found a dirty hack to fix this: put this line in transpile.js (after `options.noResolve = true;` line):
`options.noEmit = false;`

**Comment by colinbate** [comment]
Hi @hhyyrylainen I've just published a new version (2.1.0) which references TypeScript 2.1.* and fixes this issue.

If you need a fix for this which supports 2.0.* then let me know.

**Comment by hhyyrylainen** [comment]
I have now tested the new version and it works great. 
Thanks for taking the time to fix this.
